### ISSUE TITLE
Edge-case of "reprint" characteristic with Weatherlight/Portal

### ISSUE THREAD

**Issue by NilsEnevoldsen** [scryfall/feedback#188]
There are six cards in _Weatherlight_ that are marked as reprints of _Portal_ cards: https://scryfall.com/search?q=e:wth+in:por+is:reprint

While technically _Portal_ came out a month before _Weatherlight_, I think the _Weatherlight_ printings should be considered the originals for most intents and purposes. For example, MTG Salvation lists these six as reprints in _Portal_: https://mtg.gamepedia.com/Portal#Reprinted_cards

There are many cards in _Portal_ that are reprints from the _Mirage_ block, but these are the only cards in _Weatherlight_ that are marked as reprints: https://scryfall.com/search?q=e:wth+is:reprint. There are no reprints at all in _Visions_, which I take as evidence that these six _Weatherlight_ cards were intended to be the originals: https://scryfall.com/search?q=e:vis+is:reprint.

**Comment by silasary** [comment]
Note that this something that is already done in other places, such as the Fall Duel Decks.

Cards like Jeskai Elder (DDN, KTK), Contagion Clasp (DDF, SOM) and [30 others](http://mtg.wtf/card?q=st%3Add+-is%3Areprint) were 'reprinted' from the upcoming set as a way to gather hype for the new block.

Scryfall already treats the Duel Decks as the reprints, even though they are released roughly a month earlier.

**Comment by csuhta** [comment]
The reason this is happening (and related to #196) is we have Portal marked as a `starter` set, and `starter` sets are included when calculating the origin of a card (`duel_deck` sets are not). 

Because Portal came before Weatherlight, the database thinks Weatherlight is reprinting the Portal cards.

I guess we need to create some kind of manual override for this situation.

**Comment by NilsEnevoldsen** [comment]
Yes, that seems like the right strategy to me. Guess origin by set type and date; manually override edge cases.

**Comment by csuhta** [comment]
This is also now an issue with [Sword of Dungeons and Dragons](https://scryfall.com/card/h17/3)
